clients/privacy-center/package.json

```diff
@@ -12,7 +12,7 @@
     "format": "prettier --write types/ theme/ pages/ config/ components/ __tests__/",
     "format:ci": "prettier --check types/ theme/ pages/ config/ components/ __tests__/",
     "test": "jest --watch",
-    "test:ci": "jest",
+    "test:ci": "tsc --noEmit && jest",
```

Comment:
Jest just runs babel transformations without checking, so we need this to actually check them. There isn't a handy way to do this with `--watch`.